File: LiveVideoSDK/VIMS.LiveVideoSDK/VIMS.LiveVideoSDK.Model/ImageModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VIMS.LiveVideoSDK.Model
{
    public class ImageModel :BaseFile
    {
        /// <summary>
        ///  图片地址
        /// </summary>
        public string ImageUrl { get; set; }
        public string FileId { get; set; }
    }
}

File: LiveVideoApp/ptt/src/main/java/com/aimissu/ptt/entity/im/ImageModel.java
package com.aimissu.ptt.entity.im;

public class ImageModel  extends BaseFile
    {
        /// <summary>
        ///  图片地址
        /// </summary>
        public String ImageUrl ;
        public String FileId ;

        @Override
        public String toString() {
            return "ImageModel{" +
                    "ImageUrl='" + ImageUrl + '\'' +
                    ", FileId='" + FileId + '\'' +
                    ", FileName='" + FileName + '\'' +
                    ", FileCode='" + FileCode + '\'' +
                    ", FileSize='" + FileSize + '\'' +
                    '}';
        }
    }